modules/nf-core/kallisto/quant/main.nf

```diff
@@ -9,24 +9,22 @@ process KALLISTO_QUANT {
 
     input:
     tuple val(meta), path(reads)
-    path index
+    tuple val(meta2), path(index)
```

Comment:
We plan on pushing these changes to the upstream nf-core/module too right?

Reply:
Actually someone already did, but didn't correct usage here.